src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -19,27 +19,35 @@ public class CommandResult {
     /** The application should exit. */
     private final boolean exit;
 
+    /** The application should show the history. */
+    private final boolean showHistory;
```

Comment:
Not sure if you want to rename this variable to sound more like a boolean?

Reply:
I think maybe should keep because AB3 used this naming convention for showHelp too. but naming it to a boolean sounding variable makes sense too

Reply:
Yas, I keep this to follow the overall naming convention

Reply:
Yep sure that make sense